csunplugged/templates/resources/treasure-hunt.html

```diff
@@ -0,0 +1,36 @@
+{% extends "resources/base-resource.html" %}
+
+<!-- Name of resource -->
+{% block name %}{{ resource.name }}{% endblock name %}
```

Comment:
Was to match usage of parent, see base template. We could always write as block as whitespace is irrelevant?